lib/rebaser-server/src/serial_dvu_task.rs

```diff
@@ -156,38 +152,4 @@ impl SerialDvuTask {
 
         Ok(())
     }
-
-    async fn maybe_run_initial_dvu(&self) -> Result<()> {
-        let builder = self.ctx_builder.clone();
-        let ctx = builder
-            .build_for_change_set_as_system(self.workspace_id, self.change_set_id, None)
-            .await?;
-        let change_set = ChangeSet::get_by_id(&ctx, ctx.change_set_id()).await?;
-
-        if change_set.status == ChangeSetStatus::Abandoned {
-            warn!("Trying to enqueue DVU for abandoned change set. Returning early.");
-            return Ok(());
-        }
-
-        if ctx
-            .workspace_snapshot()?
-            .has_dependent_value_roots()
-            .await?
-        {
-            metric!(counter.serial_dvu_task.initial_dvu_running = 1);
-
-            info!(
-                task = Self::NAME,
-                service.instance.id = self.metadata.instance_id(),
-                si.workspace.id = %self.workspace_id,
-                si.change_set.id = %self.change_set_id,
-                "enqueuing *initial* dependent_values_update",
-            );
-            ctx.enqueue_dependent_values_update().await?;
-            ctx.blocking_commit_no_rebase().await?;
-            metric!(counter.serial_dvu_task.initial_dvu_running = -1);
-        }
-
-        Ok(())
-    }
```

Comment:
I was a bit surprised of the differences between this now-removed `maybe_run_initial_dvu` and `run_dvu` functions. At least there's no difference now, now that there's only one